src/org/centos/contra/Infra/Utils.groovy

```diff
@@ -125,23 +125,30 @@ def createBeakerInstances(HashMap<String,String>beaker_data){
         if (instance.keyvalue) {instance.keyvalue.each{ entry -> keyvalue << entry} }
         instance.keyvalue = keyvalue.toSet() as ArrayList<String>
 
-        if (instance.count){
-            String base_name = instance.name
-            for (i in 1..instance.count){
-                instance.name = "${base_name}_${i}"
+        if (instance.count) {
+            String name = instance.name ?: UUID.randomUUID().toString()
+            for (i in 1..instance.count) {
+                // For some reason, trying to create this string using string interpolation results
+                // in a JSON object instead of a String. I have no explaination for this, so I have
+                // altered the String create to use the '+' operator because this works as expected.
+                instance.name = name + "_${i}"
```

Comment:
I'd like to see this as:
`"${name}_${i}"`
for consistency

Reply:
Me too. As noted in the comment, for some reason interpolating the name object resulted in instance.name becoming a JSON object instead of a String. I will test is this is still the case, but I have no explanation for why this was happening during development.

Reply:
If that's happening, then something is wrong somewhere, better to fix than to work around.

Reply:
Went back in an restructured it like the other functions, with a trailing `.toString()` which I am convinced shouldn't be affecting the results of a gstring, but it all seems to be working with that configuration, so I'm going to close this as resolved.